### PR TITLE
PR for PageHeaderComponent, StudentStatsListView, StudentsStatsCard and miscellaneous basic configurations

### PR DESCRIPTION
**Pull request by Rushikesh-Sonawane99**
## Summary by CodeRabbit

- **New Features**
	- Introduced a `PageHeaderComponent` for enhanced navigation, featuring language selection and quick access to user account settings.
	- Added a `StudentStatsCard` component to display student statistics in a visually appealing card format.
	- Launched a `StudentsStatsListView` to present student statistics in an organized and styled layout.
- **Refactor**
	- Updated the `App` component structure for improved readability and performance.
	- Enhanced code formatting in the `CustomButton` component for consistency.
- **Style**
	- Integrated new styles to support the latest component additions and updates.

### CODE REVIEW

File: src/components/PageHeaderComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+  return (
+    <Stack
+      sx={{ minWidth: 360 }}
+      direction="row"
+      justifyContent={"space-between"}
+      alignItems="center"
+      bgcolor="white"
+      padding={'1rem'}
+      borderBottom={'2px solid #4D4639'}
```
Comment (itsvick) [inline src/components/PageHeaderComponent.tsx:31]: @Rushikesh-Sonawane99 get color from the themes

File: src/components/PageHeaderComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    >
+      <FormControl>
+        <InputLabel id="demo-simple-select-label">Language</InputLabel>
```
Comment (itsvick) [inline src/components/PageHeaderComponent.tsx:34]: @Rushikesh-Sonawane99 add language constant

Comment (itsvick) [reply src/components/PageHeaderComponent.tsx:34]: @Rushikesh-Sonawane99 remove id if not required

File: src/components/PageHeaderComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        <Select
+          labelId="demo-simple-select-label"
+          id="demo-simple-select"
+          value={age}
+          label="Age"
+          style={{ borderRadius: "20px", color:'#4D4639', width: "7rem", height: "1.625 rem" }}
+          onChange={handleChange}
+        >
+          <MenuItem value="English">EN</MenuItem>
```
Comment (itsvick) [inline src/components/PageHeaderComponent.tsx:43]: @Rushikesh-Sonawane99 Get list from the configuration file

File: src/components/PageHeaderComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+          <MenuItem value="Hindi">HI</MenuItem>
+        </Select>
+      </FormControl>
+      <Box sx={{ margin: "0 auto" }}>
+        <svg
+          width="100"
+          height="100"
+          viewBox="0 0 44 40"
+          fill="none"
```
[the remark this answers is not in the document]
Comment (Rushikesh-Sonawane99) [reply src/components/PageHeaderComponent.tsx:52]: @itsvick Added svg in public folder

File: src/components/PageHeaderComponent.tsx (hunk 2 above)
Comment (Rushikesh-Sonawane99) [reply src/components/PageHeaderComponent.tsx:34]: > @Rushikesh-Sonawane99 add language constant

Added language constants

File: src/components/PageHeaderComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            height: "1.625 rem",
+          }}
+          onChange={handleChange}
+        >
+          <MenuItem value="English">EN</MenuItem>
+          <MenuItem value="Hindi">HI</MenuItem>
+        </Select>
+      </FormControl>
+      <Box sx={{ margin: "0 auto" }}>
+        <img src="../../public/prathamlogo.svg" alt="PrathamLogo" />
```
Comment (itsvick) [inline src/components/PageHeaderComponent.tsx:57]: @Rushikesh-Sonawane99 change name of the component, path, remove prathamLogo name

File: src/components/StudentsStatsListView.tsx [hunk 6]
```diff
@@ -0,0 +1,83 @@
+import React from "react";
+
+import { Box, Link, Stack, Typography } from "@mui/material";
```
Comment (itsvick) [inline src/components/StudentsStatsListView.tsx:3]: @Rushikesh-Sonawane99 rename component to StudentsStatsList

File: src/components/StudentsStatsListView.tsx [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+interface StudentsStatsListViewProps {
+  studentName: string;
+  label1: string;
+  value1: string;
+  label2: string;
+  value2: string;
+}
+
+const StudentsStatsListView: React.FC<StudentsStatsListViewProps> = ({
+  studentName,
```
Comment (itsvick) [inline src/components/StudentsStatsListView.tsx:13]: @Rushikesh-Sonawane99 studentName can be changed to just `name`

File: src/translations/en.json [hunk 8]
```diff
@@ -1,8 +1,9 @@
 {
-
-    "LOGIN_PAGE": {
-        "USERNAME": "Username",
-        "PASSWORD": "Password"
-       
-      }
+  "COMMON": {
+    "LANG": "Language"
```
Comment (itsvick) [inline src/translations/en.json:3]: @Rushikesh-Sonawane99 make it `LANGUAGE`

File: src/components/StudentsStatsListView.tsx (hunk 6 above)
Comment (Rushikesh-Sonawane99) [reply src/components/StudentsStatsListView.tsx]: Renamed component

File: src/components/StudentsStatsListView.tsx (hunk 7 above)
Comment (Rushikesh-Sonawane99) [reply src/components/StudentsStatsListView.tsx]: RenamedComponent

File: src/translations/en.json (hunk 8 above)
Comment (Rushikesh-Sonawane99) [reply src/translations/en.json]: Changed to LANGUAGE

**Pull request merged**